src/Common/src/Common/Reflection/ReflectionHelpers.cs

```diff
@@ -30,6 +33,87 @@ public static Assembly FindAssembly(string name)
             return null;
         }
 
+        /// <summary>
```

Comment:
@dtillman There are some helpers in here that might also be interesting in relation to some of the streams work